src/sonic-config-engine/minigraph.py

```diff
@@ -274,6 +274,10 @@ def parse_cpg(cpg, hname):
             for router in child.findall(str(QName(ns1, "BGPRouterDeclaration"))):
                 asn = router.find(str(QName(ns1, "ASN"))).text
                 hostname = router.find(str(QName(ns1, "Hostname"))).text
+                if router.find(str(QName(ns1, "RRClient"))):
```

Comment:
can you add this in the sample minigraph and add unit test as well, to make sure later no one will break this feature.

Reply:
Ok let me do that and repost.

Reply:
still worth to add the sample configuration in the minigraph, so that later taoyu know how to move it to the config db.